mlrun/__main__.py

```diff
@@ -761,11 +761,30 @@ def get(kind, name, selector, namespace, uid, project, tag, db, extra_args):
 @click.option("--verbose", is_flag=True, help="verbose log")
 @click.option("--background", "-b", is_flag=True, help="run in background process")
 @click.option("--artifact-path", "-a", help="default artifact path")
-def db(port, dirpath, dsn, logs_path, data_volume, verbose, background, artifact_path):
+@click.option(
+    "--env-file",
+    default="",
+    is_flag=False,
+    flag_value="$",
+    help="update the specified mlrun .env file (if value not provided defaults to '~/.mlrun.env')",
+)
```

Comment:
At first glance looking at the param it seems like I can provide an override env file that the config will read from but this is not as an override for the default env file mlrun config is looking at, not fully understand the use of that rather than update the pid / api url afterwards...